applications/emulated_peripherals/gpio/prj.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_IPC_SERVICE=y
+CONFIG_IPC_SERVICE_BACKEND_ICMSG=y
```

Comment:
Is it needed, if backed is selected automatically ([this](https://github.com/nrfconnect/sdk-nrf/pull/16393#discussion_r1672607852)) ?

Reply:
Apparently it is necessary.

Reply:
That's curious. I guess we should investigate why the backend is not enabled by default based on the device tree. Is something disabling it?